tools-overview.md

```diff
@@ -0,0 +1,349 @@
+
+For example, you can create multiple LUIS intents in a single .lu file as follows: 
+
+```LUDown
+# Greeting
+- Hi
+- Hello
+- Good morning
+- Good evening
+
+# Help
+- help
+- I need help
+- please help
+```
+
+### QnA pairs with LUDown
+
+The .lu file format supports also QnA pairs using the following notation: 
+
+```LUDown
+> comment
+### ? question ?
+  ```markdown
+    answer
+  ```
+
+The LUDown tool will automatically separate question and answers into a qnamaker JSON file that you can then use to create your new [QnaMaker.ai](http://qnamaker.ai) knowledge base.
+
+```LUDown
+### ? How do I change the default message for QnA Maker?
+  ```markdown
+  You can change the default message if you use the QnAMakerDialog. 
+  See [this link](https://docs.botframework.com/en-us/azure-bot-service/templates/qnamaker/#navtitle) for details. 
+  ```
+
+You can also add multiple questions to the same answer by simply adding new lines of variations of questions for a single answer. 
+
+```LUDown
+### ? What is your name?
+- What should I call you?
+  ```markdown
+    I'm the echoBot! Nice to meet you.
+  ```
+
+### Generating .json models with LUDown
+
+After you've defined LUIS or QnA language components in the .lu format, you can publish out to a LUIS .json, QnA .json, or QnA .tsv file. When run, the LUDown tool will look for any .lu files within the same working directory to parse. Since the LUDown tool can target both LUIS or QnA with .lu files, we simply need to specify which language service to generate for, using the general command **ludown parse <Service> -- in <luFile>**. 
+
+In our sample working directory, we have two .lu files to parse, '1.lu' to create LUIS model, and 'qna1.lu' to create a QnA knowledge base.
+
+### Generate LUIS .json models
+
+To generate a LUIS model using LUDown, in your current working directory simply enter the following:
+
+```shell
+ludown parse ToLuis --in <luFile> 
+```
+
+### Generate QnA Knowledge Base
+
+Similarly, to create a QnA knowledge base, you only need to change the parse target. 
+
+```shell
+ludown parse ToQna --in <luFile> 
+```
+
+The resulting JSON files can be consumed by LUIS and QnA either through their respective portals, or via the new CLI tools. 
+
+See [here](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/Ludown) to learn more about the Ludown CLI.
+
+## Keep track of service references using bot file
+
+The new [MSBot](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/MSBot) tool allows you to create a **.bot** file, which stores metadata about different services your bot consumes, all in one location. This file also enables your bot to connect to these services from the CLI. The tool is available as an npm module, to install it run:
```

Comment:
**You can create a .bot file using the MSBot tool. The .bot file stores metadata about services your bot consumes. The bot can connect to services stored in the .bot file using CLI tools.**